twistr/events.go

```diff
@@ -524,6 +548,18 @@ func PlayArmsRace(s *State, player Aff) {
 	   Operations Track, that player receives 1 VP. If the phasing player has a
 	   higher value than their opponent, and has met the “required” amount, on the
 	   Military Operations Track, that player receives 3 VP instead. */
+	playerMilOps := s.MilOps[player]
+	oppMilOps := s.MilOps[player.Opp()]
+	// mil ops has to be greater than or equal to defcon to meet "required" amount
+	if playerMilOps > oppMilOps {
```

Comment:
nit/suggestion: I really prefer switch statements for anything like this:

    switch {
    case playerMilOps > oppMilOps && playerMilOps >= s.Defcon:
        s.GainVP(player, 3)
    case playerMilOps > oppMilOps:
        s.GainVP(player, 1)
    }